From 8c5ea6383f157beceaec2c593660921f63baa24f Mon Sep 17 00:00:00 2001
From: Bruno Martins <bgcngm@gmail.com>
Date: Mon, 5 Nov 2018 11:09:12 +0000
Subject: [PATCH 6/6] lineage: Enable Exchange

 * Exchange in conjunction with stock Email app is working perfectly fine.

Change-Id: I066a8d90e6880b540aab73bc8271567eb6eebae9
---
 snippets/lineage.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/snippets/lineage.xml b/snippets/lineage.xml
index aae0b27..6c03f45 100644
--- a/snippets/lineage.xml
+++ b/snippets/lineage.xml
@@ -34,7 +34,7 @@
   <project path="packages/apps/AppDrawer" name="LineageOS/android_packages_apps_AppDrawer" />
   <project path="packages/apps/AudioFX" name="LineageOS/android_packages_apps_AudioFX" />
   <project path="packages/apps/Eleven" name="LineageOS/android_packages_apps_Eleven" />
-  <!--<project path="packages/apps/Exchange" name="LineageOS/android_packages_apps_Exchange" />-->
+  <project path="packages/apps/Exchange" name="LineageOS/android_packages_apps_Exchange" />
   <project path="packages/apps/FMRadio" name="LineageOS/android_packages_apps_FMRadio" />
   <project path="packages/apps/Jelly" name="LineageOS/android_packages_apps_Jelly" />
   <project path="packages/apps/LineageCustomizer" name="LineageOS/android_packages_apps_LineageCustomizer" />
-- 
2.17.1

